hdl/modules/fofb_sys_id/cheby/wb_fofb_sys_id_regs.h

```diff
@@ -39,6 +119,88 @@ positions. The valid range of this register is [0, 255].
     /* padding to: 4 words */
     uint8_t __padding_1[3];
   } bpm_pos_flatenizer;
+
+  /* padding to: 4096 words */
+  uint32_t __padding_0[4094];
+
+  /* [0x4000]: BLOCK Interface to PRBS-related regs */
+  struct prbs {
+    /* [0x0]: REG (rw) General PRBS control register
+ */
+    uint32_t ctl;
+
+    /* [0x4]: REG (rw) Duration of each PRBS step in FOFB cycles (max: 0x03FF)
+
+write 0x0000: new PRBS iteration at each FOFB cycle
+write 0x0001: new PRBS iteration at each 2 FOFB cycles
+...
+write 0x03FF: new PRBS iteration at each 1024 FOFB cycles
+ */
+    uint16_t step_duration;
+
+    /* padding to: 8 words */
+    uint8_t __padding_0[2];
+
+    /* [0x8]: REG (rw) Length of internal LFSR (max: 0x1E)
+Together with step_duration, defines the duration of PRBS,
+which is given by [2^(lfsr_length + 2) - 1]*(step_duration + 1)
+
+write 0x00: set LFSR length to 2
+write 0x01: set LFSR length to 3
+...
+write 0x1E: set LFSR length to 32
+ */
+    uint8_t lfsr_length;
+
+    /* padding to: 4096 words */
+    uint8_t __padding_1[4087];
+
+    /* [0x1000]: BLOCK Interface to prbs_bpm_pos_distort regs */
+    struct bpm_pos_distort {
+      /* [0x0]: REG (rw) PRBS-based distortion for BPM positions control register
+ */
+      uint32_t ctl;
+
+      /* padding to: 512 words */
+      uint32_t __padding_0[511];
+
+      /* [0x800]: MEMORY Distortion levels RAM */
+      struct distort_ram {
+        /* [0x0]: REG (rw) Two S16FP0-bit distortion levels, one for each PRBS
+value.
+
+15 - 0: distortion level for PRBS value 0
+31 - 16: distortion level for PRBS value 1
+ */
+        uint32_t levels;
```

Comment:
Could we take advantage of the smaller registers here to have two `uint16_t` values?